File: core/src/main/java/by/itechart/counterparty/dto/CounterpartyDto.java
package by.itechart.counterparty.dto;

import by.itechart.common.dto.AddressDto;
import by.itechart.counterparty.enums.CounterpartyType;
import lombok.Data;

@Data
public class CounterpartyDto {
    private Long id;
    private AddressDto address;
    private String name;
    private CounterpartyType counterpartyType;
    private String taxNumber;
}

File: web-app/online-warehouse/src/app/counterparty/dto/counterparty.dto.ts
import {CounterpartyTypeEnum} from './enum/counterparty-type.enum';
import {AddressDto} from "../../shared/address/dto/address.dto";

export class CounterpartyDto {
  id: number;
  address: AddressDto;
  name: string;
  counterpartyType: CounterpartyTypeEnum;
  taxNumber: string;


  // constructor(id: number, address: AddressDto, name: string, counterpartyType: CounterpartyTypeEnum, taxNumber: string) {
  //   this.id = id;
  //   this.address = address;
  //   this.name = name;
  //   this.counterpartyType = counterpartyType;
  //   this.taxNumber = taxNumber;
  // }
}
